examples/metrics/kafka-cruise-control-metrics.yaml

```diff
@@ -36,7 +36,7 @@ spec:
       valueFrom:
         configMapKeyRef:
           name: cruise-control-metrics
-          key: metrics-config.yml
+          key: metrics-config.json
```

Comment:
Same as above ... I will not repeat it anymore, it should apply to all examples.